src/pages/Rooms.tsx

```diff
@@ -62,7 +62,7 @@ export default function Rooms({
           {studentGroups.map((group, index) => {
             return (
               <li key={rooms[index]}>
-                <b>{rooms[index]}</b>
+                <RoomNameStyled>{rooms[index]}</RoomNameStyled>
```

Comment:
```suggestion
                <RoomName>{rooms[index]}</RoomName>
```
No need for the Styled postfix as we have no naming conflict.